mindmeld/components/question_answerer.py

```diff
@@ -227,6 +236,23 @@ def load_kb(self, index_name, data_file, **kwargs):
             raise ValueError(msg)
         self._load_kb(index_name, data_file, **kwargs)
 
+    def dump(self, pkl_path):
+        folder = os.path.dirname(pkl_path)
+        os.makedirs(folder, exist_ok=True)
+        self._dump(pkl_path)
+
+        metadata = {
+            "app_path": self.app_path,
+            "app_namespace": self.app_namespace,
+            "resource_loader": self.resource_loader,
```

Comment:
With the recent refactor of the QA components, why do we have a reference to the resource loader in the QA classes in the first place?